client/src/components/Car/index.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+class Car  extends Component {
+  constructor(props) {
```

Comment:
There is no need for the `Car` component to be class. since it only renders the data it received which would be passed through the `props`